frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -127,17 +128,48 @@ export const detectOCS: FeatureDetector = async (dispatch) => {
         (sc: StorageClusterKind) => sc.status.phase !== 'Ignored',
       );
       const isInternal = _.isEmpty(storageCluster?.spec?.externalStorage);
-      dispatch(setFlag(OCS_FLAG, true));
       dispatch(setFlag(OCS_CONVERGED_FLAG, isInternal));
       dispatch(setFlag(OCS_INDEPENDENT_FLAG, !isInternal));
+      dispatch(setFlag(OCS_FLAG, true));
     }
   } catch (error) {
+    dispatch(setFlag(OCS_FLAG, false));
     dispatch(setFlag(OCS_CONVERGED_FLAG, false));
     dispatch(setFlag(OCS_INDEPENDENT_FLAG, false));
-    dispatch(setFlag(OCS_FLAG, false));
   }
 };
 
+export const detectComponents: FeatureDetector = async (dispatch) => {
+  let id = null;
```

Comment:
since its only for noobaa.
```suggestion
  let hasNoobaa = null;
```

Reply:
it's it id of the returned `setInterval` job.

Reply:
My request was to specify that its for interval job checking for noobaa only.
Since I can see when noobaa is found its getting unset.

Reply:
I would prefer `id` because that is what it stores. The `id` of the task.